### ISSUE TITLE
[FEA]document integration tests common options

### ISSUE THREAD

**Issue by tgravescs** [NVIDIA/spark-rapids#814]
**Is your feature request related to a problem? Please describe.**
it would be nice if our integration test document had some of the commonly used arguments:
pytest.TEST_TAGS
how do you specify just one test file like spark-submit run-tests.py ./src/main/python/parquet_test.py -k (both individual test file and how the -k can be used to match individual tests within that file 

it might also be nice to mention the timezone thing as I think many people run into that:
--conf spark.driver.extraJavaOptions=-Duser.timezone=GMT --conf spark.sql.session.timeZone=UTC --conf spark.executor.extraJavaOptions=-Duser.timezone=GMT

Basically common stuff that people use

**Comment by jlowe** [comment]
We may want to add a section to `CONTRIBUTING.md` on how to run the tests in general.
